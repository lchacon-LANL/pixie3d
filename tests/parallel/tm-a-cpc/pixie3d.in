 &datin
    ilevel   = 0

    nxd      = 32
    nyd      = 32
    nzd      = 1

    numerical_grid = f

    xmax     = 1.
    ymax     = 4.
    zmax     = 1.

    bcs      = 'def','def','per','per','per','per'

    eta      = 1d-2
    nu       = 1d-3
    dd       = 0d-1
    chi      = 0d-2
    
    equil    = 'tmcar'
    coords   = 'car'
    gparams  = -0.05,0d0,0d0,0d0,0d0

    dlambda  = .2
    rshear   = 1.
    vperflow = 0.

    dt       = 1d-2
    tmax     = 1d0
    dstep    = 1d-2

    pvy      = 1d-3
    odd      = t

    cnfactor = 0.5

    restart  = f
    timecorr = f
    predictor= f

    adiabatic= t
    source   = t

    precon   = 'id'
    mgtol    = 1d-5
    maxvcyc  = 1
    nsweep   = 4
    precpass = 1

    iguess   = 0

    maxksp   = 20
    maxitgm  = 20
    tolgm    = 8d-1
    rtol     = 1d-4
    method   = 1
    mf_eps   = 1d-6

    solenoidal = t
    nc_eom_f   = f
    nc_eom_v   = f

    k_si     = 0.0

    debug    = f
    debug_it = 1

 /
 &graphdef
    sel_diag  = 2,12,15,16,17,18,19,20,21
    sel_graph = 1,2,3,4,5,6,7,8,11
    dplot = 1d-2
    prof_conf = 1,'x',1,10,1
    cont_conf = 3,'x','y',1,1,1
    clean = f
 /


    sel_graph = 1,2,3,4,5,6,7,8,14
    sel_graph = 1,8,14,15,16,17,18,19,20
    sel_graph = 1,8,11,14,-15,17,-18,20,0

    equil    = 'tmcar'
    coords   = 'car'

    maxitnwt = 1
    tolgm    = 1d-4
    rtol     = 1d-3
