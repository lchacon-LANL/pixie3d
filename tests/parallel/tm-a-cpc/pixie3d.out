Processor:   0; Grid limits:    1   16    1   32    1    1
Processor:   1; Grid limits:   17   32    1   32    1    1
 
 ***********************************************
 ****** PIXIE3D Extended MHD code, V. 0.9 ******
 ******       Developer: L. Chacon        ******
 ******  Los Alamos National Laboratory   ******
 ******       Los Alamos, NM 87545        ******
 ******           LA-CC 07-005            ******
 ***********************************************

  Grid mesh:................   32x   32x    1
  Number of grid levels:....    3,    4,    1
  Time correction...........   F
  External source...........   T
  Numerical grid............   F
  Advection scheme..........   ZIP    
  TS scheme................. Theta    
 
   itime    time     dt    vx_max vy_max vz_max GMRES  Newton  CN factor
      1    0.010  1.0E-02   0.000  0.001  0.000     4     4     0.500
      2    0.020  1.0E-02   0.000  0.001  0.000     4     4     0.500
      3    0.030  1.0E-02   0.000  0.001  0.000     4     4     0.500
      4    0.040  1.0E-02   0.000  0.001  0.000     4     4     0.500
      5    0.050  1.0E-02   0.000  0.001  0.000     4     4     0.500
      6    0.060  1.0E-02   0.000  0.001  0.000     4     4     0.500
      7    0.070  1.0E-02   0.000  0.001  0.000     4     4     0.500
      8    0.080  1.0E-02   0.000  0.001  0.000     4     4     0.500
      9    0.090  1.0E-02   0.000  0.001  0.000     4     4     0.500
     10    0.100  1.0E-02   0.000  0.001  0.000     4     4     0.500

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt  Whst/GMRES
   10       4.0        4.0        1.0        0.0
